flyway-mysql57/Dockerfile.arm64

```diff
@@ -0,0 +1,24 @@
+FROM adoptopenjdk:8-jdk-hotspot-bionic
+
+LABEL version="5.7.21"
```

Comment:
@tshinowpub 
I think `[flyway-version]-[mysql-version]` is better.
e.g. chatwork/flyway-mysql57:3.1-5.7.21

Reply:
Changed the version of the k tag to support databases other than MySQL, like chatwork/flyway:3.1-mysql5.6.21.

https://github.com/chatwork/dockerfiles/pull/1011/commits/6f9baa4315b731ff499ba26a0738db7053e535c4